class-4-js-basics/ziada-nidal-js-function-intro/index.js

```diff
@@ -0,0 +1,24 @@
+// 1st function
+
+function convertMinutes (integer) {
+  const seconds = integer * 60
+  return seconds
```

Comment:
could u delete this folder from your PR I dunno how it got there bu there's already a version of it in the main repo